meson.build

```diff
@@ -0,0 +1,30 @@
+project(
+    'perun',
+    'c',
+    version: '0.21.6',
```

Comment:
Note to @tfiedor: this might need updating based on merge order of this and #148 .

Reply:
Done.